src/modelAnalysis/rumba/rumba.m

```diff
@@ -0,0 +1,294 @@
+function [RUMBA_outputs, UpRegulated, DownRegulated, MetConnectivity1, MetConnectivity2] = rumba(model1, model2, completeModel, sampling, maxMetConn, RxnsOfInterest, GenesOfInterest, NormalizePointsOption, PValCuttoff, MaxNumPoints, LoopRxnsToIgnore, verboseTag)
+% RUMBA predicts which reactions significantly change their

```

Comment:
@tpfau, yes if you could add the inputparser will be great because I never used and it is far from straightforward for me how to do it efficiently... the 3 required inputs are model1, model2 and complete model,  all the other are optional
thanks